prep-javascript-loops/main.js

```diff
@@ -0,0 +1,67 @@
+// EXERCISE 1
+function whileLoop1() {
+  const Array = [];
```

Comment:
you should not use capital letters for variables unless they are a specific type of function.

Reply:
I have fixed the issue, Thank you